lib/paymentrails/Configuration.rb

```diff
@@ -19,13 +19,13 @@ def initialize(publicKey, privateKey, environment = 'production', proxy_uri: nil
     def apiBase
       case environment
       when 'production'
-        'https://api.paymentrails.com'
+        'https://api.trolley.com'
       when 'development'
         'https://api.railz.io'
```

Comment:
same as [here](https://github.com/PaymentRails/ruby-sdk/pull/50/files#r1122511507)

Reply:
Yeah, same as there, we're tackling this in the Gap Update task to remove environments 😆